app/assets/stylesheets/questions.css.scss

```diff
@@ -1,48 +1,56 @@
-.all-mine-other-filter {
-  margin-right: 25px;  // aligns with right side of question list
-}
-
-// (All-mine-filter relocation at different screen sizes)
-// On small screens (bootstrap xs) show all-mine-other filter
-// just above the question list, since it wraps onto its own .row
-#all-mine-filter-container-xs {
+// On small screens (bootstrap xs) show-small-screen and replace with show-large-screen otherwise
+#show-small-screen {
```

Comment:
Can we keep this file alphabetical? (roughly) -- e.g. have all-mine-other (a-) entries at the top, move these s-related entries down, etc -- that makes it easier to look for things.